models/mstable/ethereum/mstable_ethereum_schema.yml

```diff
@@ -0,0 +1,93 @@
+version: 2
+
+models:
+  - name: mstable_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: mstable
+      contributors: ripple3
+    config:
+      tags: ['ethereum','mstable','trades', 'dex']
+    description: >
+        mStable trades on Ethereum.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project
+        name: project
+        description: "Project name of the DEX"
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"
+      - &block_date
+        name: block_date
+        description: "UTC event block date of each DEX trade"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DEX trade"
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the trade"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the trade"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the trade"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
```

Comment:
please add the dex generic test here on this column to validate bought address:
https://github.com/duneanalytics/spellbook/pull/1866/files#diff-8ba1908b2679efbd3dc591b730739b0c156cd7fe53a264c85c172ef2762d3205R64

for this test to run, you'll need to manually add rows to our seed file:
https://github.com/duneanalytics/spellbook/pull/1866/files#diff-09c1b2e11432d6907b7f05a0e73f46488de55e9f905df8d54602cb7173fdba5fR55

these are examples from a recently merged PR. some protocols offer an analytics page on the app, which makes it easy to find tx's which link out to etherscan. the goal is to find tx's on a third-party source and validate against the query results.

Reply:
completed this request.